src/legacy/utils.ts

```diff
@@ -398,3 +398,23 @@ export const ignoreConcurrentAsyncHandler = <Props, T, R>(
 ): ((props: Props) => (t: T) => Promise<R | void>) => {
   return uncurry(ignoreConcurrentAsync(curry(handler), additionalDelay))
 }
+
+/**
+ * Wrapper function to lodash.isEmpty that
+ * returns true if the string is empty.
+ * The lodash.isEmpty function doesn't have the typescript's safeguard signature.
+ * It will be fixed in this PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60401
+ *
+ * @summary Returns true if the value is empty: length === 0, null or undefined, else false.
+ *
+ * @param value The string to inspect
+ * @return {boolean} Returns true if the string is empty, else false.
+ *
+ * @example isEmptyString('') returns true
+ * @example isEmptyString(' ') returns false
+ * @example isEmptyString(null) returns true
+ * @example isEmptyString(undefined) returns true
+ */
+export function isEmptyString(value: string | null | undefined): value is '' | null | undefined {
```

Comment:
`isEmptyString` should validate that the value is of type `string`
`null` and `undefined` are not strings

Reply:
It's just a temporary wrapper for lodash `isEmpty` until the safeguard PR is merged. Accepting naming suggestions.